modules/dnn/test/test_halide_layers.cpp

```diff
@@ -55,10 +55,10 @@ static void test(LayerParams& params, Mat& input, Backend backendId, Target targ
     test(input, net, backendId, targetId, skipCheck, true, l1, lInf);
 }
 
-static inline testing::internal::ParamGenerator<tuple<Backend, Target> > dnnBackendsAndTargetsWithHalide()
-{
-    return dnnBackendsAndTargets(true, true, false); // OpenCV/CPU is used as reference
-}
+// static inline testing::internal::ParamGenerator<tuple<Backend, Target> > dnnBackendsAndTargetsWithHalide()
+// {
+//     return dnnBackendsAndTargets(true, true, false); // OpenCV/CPU is used as reference
+// }
 
 class Test_Halide_layers : public DNNTestLayer {};
```

Comment:
Feel free to rename to something like `Test_layers_backends` in this PR or later.

Reply:
I found there is `test_backends.cpp` which basically does the same thing testing other backends against default. I think we'd better merge `test_halide_layers.cpp` into it.